templates/vue/src/components/NodeModal.vue

```diff
@@ -257,18 +291,14 @@ export default {
     },
     canPublish() {
       if (this.loading) return false
-      if (this.type === "add") {
+      if (
+        this.type === "add" ||
+        (this.type === "edit" && this.node.status !== "publish")
+      ) {
         return (
           Helpers.hasPermission(this.parent, this.type) &&
           (!this.parent || this.parent.status !== "draft")
         )
-      } else if (this.node.status === "draft" && this.type === "edit") {
-        return this.getNeighbours(this.nodeId).some(neighbourId => {
-          let neighbour = this.getNode(neighbourId)
-          return (
-            neighbour.status !== "draft" && Helpers.hasPermission(neighbour, "add")
-          )
-        })
```

Comment:
I had said [before](https://github.com/wynnset/tapestry-wp/pull/683#discussion_r516274856) (I think it might have been missed in between my other feedback): The concept of `this.parent` does not exist within edit context. How do you decide which node is the parent if the node is connected to more than a single other node? That is why the `else if` statement was added. I think you'd need to revert so that we check ALL the neighbouring nodes and see if ANY of them are not published or accepted.

Reply:
Why was this marked as resolved? It has not been resolved...

Reply:
Sorry, I didn't actually push!